test-app/.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
I suppose we no longer need this file then?

Reply:
Very true :)